### ISSUE TITLE
Floating Dock toggle doesn’t work

### ISSUE THREAD

**Issue by colindavidr** [MTACS/Lynx2#347]
Please fill out all fields with as much detail as possible

**Describe the bug:**

**Process To Reproduce:**

**Screenshots:**

**Platform (please complete the following information):**
 - Device:
 - Jailbreak: 
    - unc0ver
    - Taurine
    - checkra1n
    - palera1n (specify rootful or rootless)
    - Dopamine
    - Xina (Only rootless version is supported)
    - Serotonin (bootstrap based)
 - iOS Version: 16.4, dopamine 2 iPhone 14 Pro

**Comment by MTACS** [comment]
This one? 
![image](https://github.com/MTACS/Lynx2/assets/13209789/29a26e54-b3c6-47d9-93b5-bbabcdb64208)

**Comment by colindavidr** [comment]
Yes.Sent from my iPhoneOn Feb 26, 2024, at 5:29 PM, D.F. ***@***.***> wrote:﻿
This one?
image.png (view on web)
